vercel.json

```diff
@@ -1,17 +0,0 @@
-{
```

Comment:
This file is needed for the playground to work. The playground is currently broken.